templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete Car</h1>
+  <p>Are you sure you wand to delete the car: {{ car }}</p>
+  <form action="" method="post">
+  {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+  </form>
```

Comment:
You should add a 'Cancel' button alongside the 'Delete' button. This 'Cancel' button should redirect the user to the previous page they were on.

Reply:
done